tangelo/toolboxes/molecular_computation/molecule.py

```diff
@@ -65,6 +65,40 @@ def molecule_to_secondquantizedmolecule(mol, basis_set="sto-3g", frozen_orbitals
     return converted_mol
 
 
+def spatial_from_spinorb(one_body_coefficients, two_body_coefficients):
+    """Function to reverse openfermion.chem.molecular_data.spinorb_from_spatial.
+
+    Args:
+        one_body_coefficients: One-body coefficients (array of 2N*2N, where N
+            is the number of molecular orbitals).
+        two_body_coefficients: Two-body coefficients (array of 2N*2N*2N*2N,
+            where N is the number of molecular orbitals).
+
+    Returns:
+        (array of floats, array of floats): One- and two-body integrals (arrays
+            of N*N and N*N*N*N elements, where N is the number of molecular
+            orbitals.
+    """
+    # Get the number of MOs = number of SOs / 2.
+    n_mos = int(one_body_coefficients.shape[0] / 2)
+
+    # Initialize Hamiltonian integrals.
+    one_body_integrals = np.zeros((n_mos, n_mos), dtype=complex)
+    two_body_integrals = np.zeros((n_mos, n_mos, n_mos, n_mos), dtype=complex)
+
+    # Loop through coefficients.
+    for p in range(n_mos):
+        for q in range(n_mos):
+            # Populate 1-body integrals.
+            one_body_integrals[p, q] = one_body_coefficients[2 * p, 2 * q]
+            # Continue looping to prepare 2-body integrals.
+            for r in range(n_mos):
+                for s in range(n_mos):
+                    two_body_integrals[p, q, r, s] = two_body_coefficients[2 * p, 2 * q + 1, 2 * r + 1, 2 * s]
```

Comment:
For readability, I suggest removing spaces around the multiplication signs.
Readability directly result from varying the number of spaces between characters: putting the same amount of spaces everywhere is not much different than putting spaces nowhere.

ex: `two_body_coefficients[2*p, 2*q + 1, 2*r + 1, 2*s]`